WEB_Q_A/client/src/app/login/login/login.component.html

```diff
@@ -0,0 +1,50 @@
+<!DOCTYPE html>
```

Comment:
Need to remove doctype, head, and body tags.
The buttons need to have method execution for login and link to register for register.
The form fields need to be bound to angular model